client/src/components/main.jsx

```diff
@@ -11,28 +11,42 @@ class Main extends React.Component {
     }
   }
 
-  // componentDidMount() {
-  //   this.setState({details: this.props.details})
-  // }
-
   render () {
-    var test = this.props.details[1]
+    var test = this.props.details[0]
```

Comment:
@SimonC989 @carl12 Should use AirBnB style guide to lint for FEC and SDC. Var, will become a thing of the past.